### DISCUSSION TITLE
Loading assets from Rails app in docs page

### DISCUSSION THREAD

**Discussion by oliverguenther** [lookbook-hq/lookbook#475] [Q&A] [answered]
Is there a documented or recommended way to get application assets into the layout that renders doc pages? In my case, I'd like our own icon font to be present in order to use that without needing to embed a component.

Of course I could override the skeleton or controller, but that would require duplicating a lot of the layout.

For now I'm working around it by simply calling a helper that loads the font in the md.erb itself.

**Comment by allmarkedup** [comment, marked as answer]
Hey @oliverguenther, at the moment there isn't I'm afraid 😕

The reason for this is that if users can customise the styling via adding arbitrary CSS then it means that if any new version of Lookbook makes changes to the markup/structure/display of those documentation pages it could be a breaking change for anyone who has customised it, even if it was a minor version bump.

I realise that is not ideal and that people will probably find ways around it anyway but it's not something that Lookbook officially supports at this time. However I'd love to make the doc pages more flexible and customisable, that is definitely something that is in the backlog although quite what shape it will take I'm not sure yet.

Sorry I can't be of more help with this one!

**Reply by oliverguenther** [reply]
Thanks for the quick response! I'm getting by with providing a special css bundle that I simply load as part of the md.erb partial. This works fine, but is bit hacky of course. I'm wondering if the `user_styles` content block was written for that purpose, but I couldn't find any documentation on it.
